BowlingBall.Tests/GameFixture.cs

```diff
@@ -1,14 +1,88 @@
 using System;
 using Xunit;
-
 namespace BowlingBall.Tests
 {
     public class GameFixture
     {
+        Game game=null;

```

Comment:
use var.
Also make it private variable as you don't need it outside

Reply:
Cannot use var with global variables